fiftyone/utils/eval/regression.py

```diff
@@ -293,12 +309,27 @@ def evaluate_samples(
             _confs = confs
             _ids = ids
 
+        # Metric operators.
+        agg_metric_ops = []
+        if eval_metrics:
+            for metric_op, metric_kwargs in eval_metrics:
```

Comment:
```
Traceback (most recent call last):
  File "/home/manushree/voxel51/custom_metrics/test_regression.py", line 4, in <module>
    import fiftyone as fo
  File "/home/manushree/voxel51/fiftyone/fiftyone/__init__.py", line 31, in <module>
    from fiftyone.__public__ import *
  File "/home/manushree/voxel51/fiftyone/fiftyone/__public__.py", line 250, in <module>
    from .utils.eval.classification import (
  File "/home/manushree/voxel51/fiftyone/fiftyone/utils/eval/__init__.py", line 20, in <module>
    from .regression import (
  File "/home/manushree/voxel51/fiftyone/fiftyone/utils/eval/regression.py", line 27, in <module>
    import fiftyone.operators as foo
  File "/home/manushree/voxel51/fiftyone/fiftyone/operators/__init__.py", line 9, in <module>
    from .registry import (
  File "/home/manushree/voxel51/fiftyone/fiftyone/operators/registry.py", line 12, in <module>
    from .builtin import BUILTIN_OPERATORS, BUILTIN_PANELS
  File "/home/manushree/voxel51/fiftyone/fiftyone/operators/builtin.py", line 20, in <module>
    from fiftyone.operators.builtins.panels.model_evaluation import EvaluationPanel
  File "/home/manushree/voxel51/fiftyone/fiftyone/operators/builtins/panels/model_evaluation/__init__.py", line 14, in <module>
    from fiftyone import ViewField as F
ImportError: cannot import name 'ViewField' from partially initialized module 'fiftyone' (most likely due to a circular import) (/home/manushree/voxel51/fiftyone/fiftyone/__init__.py)
```

What is the best way to import FO operators in fiftyone/utils/eval? I would like to add operator URI instead of the operator itself in `eval_metrics`. @ritch @brimoor